libsolidity/ast/ASTAnnotations.h

```diff
@@ -148,16 +148,21 @@ struct StructDeclarationAnnotation: TypeDeclarationAnnotation
 	std::optional<bool> containsNestedMapping;
 };
 
-struct ContractDefinitionAnnotation: TypeDeclarationAnnotation, StructurallyDocumentedAnnotation
+struct SharedContractDependenciesAnnotation
+{
+	/// List of contracts this contract or function creates, i.e. which need to be compiled first.
+	/// For ContractDefinitions: Also includes all contracts from linearizedBaseContracts.
+	/// Only set for contracts or free functions
```

Comment:
```suggestion
	/// Only relevant for contracts or free functions
```
set was a bit confusing